content/en/guides/hosting/data-security/secure-storage-connector.md

```diff
@@ -356,13 +355,15 @@ Select a tab for detailed instructions.
 
 {{< tabpane text=true >}}
 {{% tab header="CoreWeave" value="coreweave" %}}
+To configure CoreWeave for instance-level BYOB, you specify the bucket name rather than the full bucket path. You are ready to [configure instance level BYOB]({{< relref "#configure-instance-level-byob" >}}).
```

Comment:
```suggestion
To configure CoreWeave for instance-level BYOB, specify the bucket name rather than the full bucket path. You are ready to [configure instance level BYOB]({{< relref "#configure-instance-level-byob" >}}).
```

Other:
The second sentence seems out of place? Maybe omit?: 
"You are ready to configure instance level BYOB."